From 31a29d922e7ae0f35d3d1f9a8e696192b49425cb Mon Sep 17 00:00:00 2001
From: Kevin Tung <kevin.tung@quantatw.com>
Date: Thu, 13 Nov 2025 19:11:44 +0800
Subject: [PATCH 17/17] PSUSensor: add support for mp2925 and mp2929

Add support for the MPS MP2925 and MP2929 voltage regulator.
The corresponding driver is available in linux-staging.git [1].

[1] https://git.kernel.org/pub/scm/linux/kernel/git/groeck/linux-staging.git/commit/?h=hwmon-next&id=a79472e30be48e6693f83d298836880b56afc5a7

Change-Id: I2116acb2cacf9e66c21d22ea0cffb39e9523b6f7
Signed-off-by: Kevin Tung <kevin.wtmh@gmail.com>
---
 src/psu/PSUSensorMain.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/psu/PSUSensorMain.cpp b/src/psu/PSUSensorMain.cpp
index d2f88c6..7db25fe 100644
--- a/src/psu/PSUSensorMain.cpp
+++ b/src/psu/PSUSensorMain.cpp
@@ -120,6 +120,8 @@ static const I2CDeviceTypeMap sensorTypes{
     {"MP2856", I2CDeviceType{"mp2856", true}},
     {"MP2857", I2CDeviceType{"mp2857", true}},
     {"MP2869", I2CDeviceType{"mp2869", true}},
+    {"MP2925", I2CDeviceType{"mp2925", true}},
+    {"MP2929", I2CDeviceType{"mp2929", true}},
     {"MP2971", I2CDeviceType{"mp2971", true}},
     {"MP2973", I2CDeviceType{"mp2973", true}},
     {"MP2975", I2CDeviceType{"mp2975", true}},
-- 
2.47.3

